protos/getresource.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package getresource;
+
+message GetResourceRequest {
+    string KbcName = 1;
+    string KbsUri = 2;
```

Comment:
I'd consider documenting all message fields. For example, is this a real URL or a "fake" one? See https://github.com/confidential-containers/attestation-agent/issues/36.